blueprints/factories/vm-factory/README.md

```diff
@@ -0,0 +1,53 @@
+# Google Cloud VM Factory
+
+This module allows creation and management of compute VM's by defining them in well formatted `yaml` files.
+
+Yaml abstraction for VM creation can simplify users onboarding compared to HCL.
+
+This factory is based on the [compute-vm module](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/compute-vm)
+
+You can create as many files as you like, the code will loop through it and create the required variables in order to execute everything accordingly.
+
+## Example
+
+### Terraform code
+
+```hcl
```

Comment:
Anita, I'm struggling to understand how this example is a factory...

Reply:
I'm not seeing the yaml referenced in the example, and I see for_each at the module level...

Reply:
A factory should take a data path, and internally parse files and manage resources. Look to the net-vpc or folder factories for examples. We also don't much like using modules inside factories, that creates too many layers of abstraction.

Reply:
I don't know if an example that does a for each on the vm module is complex enough to put in the repo... let's close this and discuss live?